R/PRISMA_flowdiagram.R

```diff
@@ -435,66 +438,28 @@ PRISMA_flowdiagram <- function (data,
-    }
-    x <- insertJS(x)
-    } else if (paste0(previous,  other) == 'FALSETRUE'){
-    insertJS <- function(plot){
-      javascript <- htmltools::HTML('
-var theDiv = document.getElementById("node1");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'497\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Identification</text>";
-var theDiv = document.getElementById("node2");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'315\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Screening</text>";
-var theDiv = document.getElementById("node3");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'100\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Included</text>";
-                              ')
-      htmlwidgets::appendContent(plot, htmlwidgets::onStaticRenderComplete(javascript))
-    }
-    x <- insertJS(x)
-  } else if (paste0(previous,  other) == 'TRUEFALSE'){
-    insertJS <- function(plot){
-      javascript <- htmltools::HTML('
-var theDiv = document.getElementById("node1");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'536\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Identification</text>";
-var theDiv = document.getElementById("node2");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'357\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Screening</text>";
-var theDiv = document.getElementById("node3");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'95\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Included</text>";
-                              ')
-      htmlwidgets::appendContent(plot, htmlwidgets::onStaticRenderComplete(javascript))
-    }
-    x <- insertJS(x)
-  } else {
-    insertJS <- function(plot){
-      javascript <- htmltools::HTML('
-var theDiv = document.getElementById("node1");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'497\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Identification</text>";
-var theDiv = document.getElementById("node2");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'315\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Screening</text>";
-var theDiv = document.getElementById("node3");
-theDiv.innerHTML += "<text text-anchor=\'middle\' style=\'transform: rotate(-90deg);\' x=\'100\' y=\'19\' font-family=\'Helvetica,sans-Serif\' font-size=\'14.00\'>Included</text>";
-                              ')
-      htmlwidgets::appendContent(plot, htmlwidgets::onStaticRenderComplete(javascript))
-    }
-    x <- insertJS(x)
-    }
+
+  insertJS_ <- function (plot) {
```

Comment:
Could you move this function outside of the function it is nested in, and add some brief documentation to it? It will just make future maintenance/testing easier!

Reply:
OK!